lib/image.ex

```diff
@@ -1297,7 +1297,7 @@ defmodule Image do
     end
   end
 
-  @spec resize(Vimage.t() | Path.t(), width :: pos_integer(), options :: Resize.resize_options()) ::
+  @spec resize(Vimage.t() | Path.t(), width :: binary(), options :: Resize.resize_options()) ::
```

Comment:
I think this fix should resolve the dialyzer error on its own, but I'll test and see.